src/Durable/OrchestrationContext.cs

```diff
@@ -30,6 +31,9 @@ public class OrchestrationContext
         [DataMember]
         internal HistoryEvent[] History { get; set; }
 
+        [DataMember]
```

Comment:
We don't expect CurrentUtcDateTime to be passed to the PowerShell worker from the host, correct? As far as I understant, this property is initialized and used by the PowerShell worker only. In this case, the `DataMember` attribute is not needed, please remove it.